game/scripts/vscripts/Sections/Chapter4/SectionOutpost.ts

```diff
@@ -158,9 +185,22 @@ function onStop() {
     }
 }
 
+function orderFilter(event: ExecuteOrderFilterEvent): boolean {
+    if (event.order_type === UnitOrder.DROP_ITEM || event.order_type === UnitOrder.MOVE_ITEM) {
+        displayDotaErrorMessage("Drop, move items are disabled in this section.")
+        return false;
+    }
+    if (event.order_type === UnitOrder.CAST_NO_TARGET && !allowUseItem) {
+        displayDotaErrorMessage("Use item will be allowed later in this section.")
```

Comment:
Consider changing to what's happening now: "Item usage is currently disabled." (or: not allowed)